src/App.css

```diff
@@ -1,4 +1,8 @@
 * {
     font-family: 'Lato', sans-serif;
     font-weight: 400;
+  }
+  .app-container {
+    width: 100%;
+    height: auto;
```

Comment:
Always check the doc first
https://getbootstrap.com/docs/5.1/layout/containers/